livekit-plugins/livekit-plugins-cartesia/livekit/plugins/cartesia/tts.py

```diff
@@ -49,6 +54,7 @@ def __init__(
         sample_rate: int = 24000,
         api_key: str | None = None,
         http_session: aiohttp.ClientSession | None = None,
+        sentence_tokenizer: tokenize.SentenceTokenizer = tokenize.basic.SentenceTokenizer(),
```

Comment:
Do we need a sentence_tokenizer when using the Cartesia API?
Can't we just send words and Cartesia directly?

Reply:
for continuations, the pacing of speech does improve a bit if we pass in sentences. alternatively, we could buffer in the first 4-5 words and then stream in words to the websocket as they come in. 

between these two approaches, what would work better within the livekit agents framework?

Reply:
ah ok, I think it is OK to keep sentences. Just wondering what is the best for Cartesia

Reply:
yep, generally slightly longer outputs have better prosody and pacing!

Reply:
For long sentences coming from an LLM. I'm wondering if it should be better to send words instead?
Seems like this could improve the time to first frame?

Reply:
We are also currently making a new request for each sentence (using the same WS). Does cartesia support multiple sentences in one single request instead (context_id)?

Reply:
Yes! So all the requests sent using one `context_id` are treated as part of a "continuation" which means that the speech for this group of requests will sound contiguous.

Reply:
> For long sentences coming from a LLM. I'm wondering if it should be better to send words instead?
Seems like this could improve the time to first frame?

Tokenizing on sentences usually leads to more natural-sounding speech. Words will work fine too, but for the first request sent to a `context_id`, we should buffer to 4-5 words.